src/svgSprite.ts

```diff
@@ -1106,6 +1106,10 @@ export const svgSprite = (): string => `
     <symbol viewBox="0 0 24 24" id="svg-trending-down">
       <path d="M16.4537 15.9159C16.5068 15.7954 16.5356 15.6657 16.5385 15.534L16.5385 8.46297C16.5385 8.33111 16.5126 8.20055 16.4621 8.07872C16.4116 7.9569 16.3377 7.84621 16.2444 7.75297C16.0561 7.56467 15.8007 7.45888 15.5344 7.45888C15.4026 7.45888 15.272 7.48485 15.1502 7.53531C15.0284 7.58577 14.9177 7.65973 14.8244 7.75297C14.6361 7.94128 14.5303 8.19667 14.5303 8.46297L14.5374 13.1228L9.17049 7.75587C8.98289 7.56833 8.72859 7.46297 8.46339 7.46297C8.19819 7.46297 7.94379 7.56833 7.75629 7.75587C7.56869 7.9434 7.46339 8.19776 7.46339 8.46297C7.46339 8.72819 7.56869 8.98254 7.75629 9.17008L13.1232 14.537L8.46339 14.53C8.33139 14.5294 8.20049 14.555 8.07849 14.6053C7.95639 14.6556 7.84549 14.7295 7.75219 14.8228C7.65879 14.9162 7.58489 15.0271 7.53459 15.1492C7.48429 15.2712 7.45869 15.402 7.45929 15.534C7.45869 15.6661 7.48429 15.7969 7.53459 15.9189C7.58489 16.041 7.65879 16.1519 7.75219 16.2453C7.84549 16.3386 7.95639 16.4125 8.07849 16.4628C8.20049 16.5131 8.33139 16.5387 8.46339 16.5381L15.5344 16.5381C15.6661 16.5353 15.7958 16.5064 15.9163 16.4533C16.1591 16.3519 16.3522 16.1588 16.4537 15.9159Z" fill="currentColor"/>
     </symbol>
+
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="#42474e" id="svg-arrow-navigate-left">
+      <path d="M17 10.9004H9.4L12.7 7.60039C13.1 7.20039 13.1 6.60039 12.7 6.20039C12.3 5.80039 11.7 5.80039 11.3 6.20039L6.3 11.2004C5.9 11.6004 5.9 12.2004 6.3 12.6004L11.3 17.6004C11.7 18.0004 12.3 18.0004 12.7 17.6004C13.1 17.2004 13.1 16.6004 12.7 16.2004L9.4 12.9004H17C17.6 12.9004 18 12.5004 18 11.9004C18 11.4004 17.6 10.9004 17 10.9004Z" fill="#42474e"/>
```

Comment:
I think it's better not to use a predefined fill here to allow for the possibility of using SVGs in other places with a different fill.